package.json

```diff
@@ -7,9 +7,10 @@
   "scripts": {
     "cli": "ts-node --require ./app/test/globals.ts --require ./app/src/cli/dev-commands-global.ts app/src/cli/main.ts",
     "test:integration": "cross-env TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe --auto-servernum -- mocha -t 30000 --require ts-node/register app/test/integration/*.ts",
-    "test:unit": "ts-node script/unit-tests.ts",
-    "test:script": "mocha -t 10000 --require ts-node/register script/changelog/test/*.ts",
-    "test": "yarn test:unit && yarn test:script && yarn test:integration",
+    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 node_modules/.bin/electron ./node_modules/jest/bin/jest --config ./jest.config.js",
+    "test:unit:cov": "yarn test:unit --coverage",
+    "test:script": "mocha -P ./tsonfig.json -t 10000 --require ts-node/register script/changelog/test/*.ts",
+    "test": "yarn test:unit --runInBand && yarn test:script && yarn test:integration",
```

Comment:
this is useful for running on CI environments, where spawning multiple processes can choke the limited resources. i put it here since i figured running all of the tests at once was mostly a CI thing and i was trying to avoid a duplicate task with just one more flag. is there a better place to put this flag for just CI?

Reply:
maybe it worth having a separate yarn test script for CI

Reply:
@shiftkey I'd like to defer a more complete solution for this to a future PR if that's ok? I think the solution is to have a separate script for this that CI uses. and/or introduce some extra configuration into our CI configs.

Reply:
I'd rather consistency between "what the user runs" and "what CI runs" so I'm happy to leave that in for now. I don't think the tests as-is will benefit from being sped up enough anyway.